src/DependencyInjection/AbstractSonataAdminExtension.php

```diff
@@ -16,7 +16,7 @@
 use Sonata\AdminBundle\FieldDescription\FieldDescriptionInterface;
 use Sonata\AdminBundle\Templating\TemplateRegistryInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Symfony\Component\DependencyInjection\Extension\Extension;
```

Comment:
I don't know if we have something to prevent about this change. but technically, this is a BC break.
The [replacing class](https://github.com/symfony/dependency-injection/blob/77c636dfd86c0b60c5d184b2fd2ddf8dd11c309c/Extension/Extension.php#L27) does not inherit the deprecated class.

Reply:
I agree with marking our own DI extension as internal. AFAIK, we haven't defined any extension point here (but we should check, I'm just talking from what I remember).

Reply:
The AbstractSonataAdminExtension is used in other bundle and could technically be used by someone else for a custom persistence bundle...
https://github.com/search?q=org%3Asonata-project%20AbstractSonataAdminExtension&type=code